.github/actions/buildkite/README.md

```diff
@@ -0,0 +1,59 @@
+## About
+
+GitHub Action to run a BuildKite pipeline using Vault
+
+___
+
+* [Usage](#usage)
+  * [Configuration](#configuration)
+* [Customizing](#customizing)
+  * [inputs](#inputs)
+
+## Usage
+
+### Configuration
+
+
+```yaml
+---
+name: Run In BuildKite
+on:
+  workflow_run:
+    workflows: [ ci ]
+    types: [ completed ]
+
+jobs:
+  build-sign:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Run BuildKite pipeline
+        uses: elastic/apm-pipeline-library/.github/actions/buildkite@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: observability-release-helm
+          buildEnvVars: '{ "slackChannel": "#my-channel", "retry": "true" }'
```

Comment:
I tested it out and it worked

<img width="331" alt="image" src="https://user-images.githubusercontent.com/2871786/215526983-e8499acf-ed00-4838-9fb6-f2010ff07c9d.png">


```
...
          waitFor: true
          printBuildLogs: true
          buildEnvVars: '{ "slackChannel": "#my-channel", "MYVAR": "my-super-value" }'
```

Reply:
This is now based on key=value multistring

<img width="351" alt="image" src="https://user-images.githubusercontent.com/2871786/215718402-e4950d7d-6b31-4db0-bc47-60c510770c22.png">


Given 

```
          buildEnvVars: |
            commit=abderg
            message=my-message
            org=my-org
            something=my super duper variable
```